port/cpl_vsi_mem.cpp

```diff
@@ -70,6 +72,38 @@
 
 //! @cond Doxygen_Suppress
 
+// szHIDDEN_DIRNAME is for files created by VSIMemGenerateUnlistedFilename(pszFilename).
+// Such files are of the form "/vsimem/.#!HIDDEN!#./{counter}/{pszFilename}"
+//
+// The high-level design constraint is that "/vsimem/.#!HIDDEN!#." acts as a
+// "side" hiearchy, but still unders the "/vsimem/" namespace, so that code
+// acting specifically on /vsimem/ files. The structure of the returned filename
```

Comment:
this reads like an unfinished sentence (and I'm not sure how to finish it).

Reply:
I meant: "so that code having special processing of filenames starting with /vsimem/ can still work"